src/zimscraperlib/zim/creator.py

```diff
@@ -146,7 +150,30 @@ def config_indexing(
         self.__indexing_configured = True
         return self
 
+    def _get_illustration_metadata(
+        self,
+        name: str,
+        value: bytes | str | datetime.datetime | datetime.date | Iterable[str],
+    ) -> str | None:
+        """Return image format for debug logging of illustration metadata"""
+        if not name.startswith("Illustration_"):
+            return None
+        if not isinstance(value, bytes):
+            return f"Unexpected image datatype, {value.__class__.__name__}"
```

Comment:
```suggestion
            return f"Unexpected image datatype, {type(value)}"
```

Reply:
This information have been lost. In such case, you _sdolved_ the comment in a different way than the suggested one and you should explain why and not resolve conversation yourself